addon-resizer/Makefile

```diff
@@ -21,7 +21,7 @@ ALL_ARCH = amd64 arm arm64 ppc64le s390x
 ARCH ?= amd64
 
 GOARM=7
-GOLANG_VERSION = 1.12.1
+GOLANG_VERSION = 1.14.1
```

Comment:
IIUC, 1.14.1 had some bug.
@mm4tt - do you remember?

In k8s we use 1.14.3 - so maybe we could stick to that version here too:
https://github.com/kubernetes/release/blob/master/images/build/cross/Makefile#L23

Reply:
That supposed to be 1.14.4, wrongly amended :-/

Changed to 1.14.3